CMakeLists.txt

```diff
@@ -995,16 +998,28 @@ list(APPEND SRC_FILES
     ${PROJECT_SOURCE_DIR}/src/mbgl/util/work_request.cpp
 )
 
-if(MLN_WITH_OPENGL)
-    message(STATUS "Configuring GL-Native with OpenGL renderer backend")
-    target_compile_definitions(
-        mbgl-core
-        PRIVATE MLN_RENDER_BACKEND_OPENGL=1
-        PUBLIC
-            "MLN_LEGACY_RENDERER=$<BOOL:${MLN_LEGACY_RENDERER}>"
-            "MLN_DRAWABLE_RENDERER=$<BOOL:${MLN_DRAWABLE_RENDERER}>"
-            "MLN_USE_UNORDERED_DENSE=$<BOOL:${MLN_USE_UNORDERED_DENSE}>"
+# if (NOT MLN_WITH_EMSCRIPTEN)
```

Comment:
I initially thought that these were giving me headaches - I will likely undo the change.